src/markdown-pages/documentation-zh_HANS.md

```diff
@@ -0,0 +1,26 @@
+---
+lang: zh_hans
+templateKey: documentation-page
+slug: documentation
+pageTitle: Project | Documentation
```

Comment:
Do you want to add placeholders for the page title (i.e. `[zh_hans] Project | Documentation`)?

Reply:
Yeah I did that for the landing page but forgot to for the documentation page - just added it